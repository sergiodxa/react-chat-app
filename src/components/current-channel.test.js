import React from "react";
import { render } from "@testing-library/react";

import CurrentChannel from "./current-channel";

test("CurrentChannel component", () => {
  const { asFragment } = render(<CurrentChannel channel="general" />);
  expect(asFragment()).toMatchSnapshot();
});
