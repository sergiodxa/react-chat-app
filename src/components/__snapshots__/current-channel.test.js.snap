// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrentChannel component 1`] = `
<DocumentFragment>
  .emotion-1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 1rem .5rem;
  box-sizing: border-box;
  height: 4rem;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-0 {
  font-size: 1.2rem;
  font-weight: 500;
}

<div
    class="emotion-1"
  >
    <strong
      class="emotion-0"
    >
      # general
    </strong>
  </div>
</DocumentFragment>
`;
